custom/icds_reports/utils/__init__.py

```diff
@@ -1846,3 +1846,10 @@ def get_deprecation_info(locations, show_test, multiple_levels=False):
 
 def get_location_replacement_name(location, field, replacement_names):
     return [replacement_names.get(loc_id, '') for loc_id in location.metadata.get(field, [])]
+
+
+def timestamp_string_to_date_string(ts_string):
+    if ts_string:
+        return ts_string[:11]
```

Comment:
This is pretty ugly. How about converting to a datetime and then pulling out the date?